docs/ExpectAPI.md

```diff
@@ -416,15 +416,15 @@ For example, this code tests that the promise rejects with reason `'octopus'`:
 ```js
 test('rejects to octopus', () => {
   // make sure to add a return statement
-  return expect(Promise.reject('octopus')).rejects.toBe('octopus');
+  return expect(Promise.reject(new Error('octopus'))).rejects.toThrow('octopus');
 });
 ```
 
 Alternatively, you can use `async/await` in combination with `.rejects`.
 
 ```js
 test('rejects to octopus', async () => {
-  await expect(Promise.reject('octopus')).rejects.toBe('octopus');
+  await expect(Promise.reject(new Error('octopus'))).rejects.toThrow('octopus');
```

Comment:
I'm wondering if we should say that `toThrow` only works in jest 21.3.0+. People copy-paste examples, and having them not working is not a good UX

Reply:
currently, it mentions
```
### `.rejects`

##### available in Jest **20.0.0+**
```

Change it to `21.3.0+`?

Reply:
Or a separate line describing rejecting promises and `toThrow`.

Reply:
@lsentkiewicz this came up in #4945, mind sending a PR with clarification? 🙂

Reply:
really painful way to discover why `rejects.toThrow` was not working on my `jest@21.2` setup

@SimenB you were right, it is probably a good idea to have a mention to it in the docs

Reply:
Can you send a PR fixing the docs?